Extensions/render/backend/mirror.lisp

```diff
@@ -3,24 +3,17 @@
 (defclass image-mirror-mixin ()
   ((image :initform nil :reader image-mirror-image)
    (image-lock :initform (clim-sys:make-lock "image"))
-   (resize-image-p :initform t :reader image-mirror-resize-image-p)
    (dirty-region :initform nil)
    (state :initform (aa:make-state))))
 
-(defmethod (setf image-mirror-image) (img (mirror image-mirror-mixin))
-  (when img
-    (with-slots (image resize-image-p) mirror
-      (setf resize-image-p nil)
-      (setf image img))))
```

Comment:
This is still used by `with-output-to-image` at this point.

Reply:
right, I will re-introduce the old handling of the image in with-output-to-image, I've missed that it is used.

Reply:
fixed (however this logic seems shady, notice that setting img to nil is a no-op)